client/packages/programs/src/JsonForms/common/components/Date.tsx

```diff
@@ -59,15 +59,13 @@ const UIComponent = (props: ControlProps) => {
               path,
               !e ? undefined : formatDateTime.customDate(e, 'yyyy-MM-dd')
             );
-            if (customError) setCustomError(undefined);
+            customErrors.remove(path);
           }}
           format="P"
           disabled={!props.enabled}
-          error={customError ?? props.errors ?? zErrors ?? ''}
+          error={props.errors ?? zErrors ?? ''}
```

Comment:
As mentioned above, the `additionalErrors` are now included in the `props.errors` field